rules-tests/Naming/Rector/Class_/RenamePropertyToMatchTypeRector/Fixture/keep_date_time.php.inc

```diff
@@ -4,7 +4,7 @@ namespace Rector\Tests\Naming\Rector\Class_\RenamePropertyToMatchTypeRector\Fixt
 
 final class KeepDateTime
 {
-    private ?DateTime $timestamp = null;
+    private ?\DateTime $timestamp = null;
```

Comment:
this was invalid fixture, that missing `\` suffix as not from use statement, and it inside namespace, so lookup of `DateTime` under fixture namespace, the extended `DateTimeInterface` already covered at 

```php
$type->isInstanceOf(ClassName::DATE_TIME_INTERFACE)->yes()
```